WebApp/autoreduce_webapp/selenium_tests/pages/help_page.py

```diff
@@ -42,6 +43,7 @@ def launch(self) -> HelpPage:
         # Navigates to / first to force a login. Check the README and
         # the "index" view for more details
         self.driver.get(configuration.get_url())
+        self.driver.get("http://localhost:0000/overview")
```

Comment:
This shouldn't hard code the URL but I don't think overview does anything special, it's `/` that we want to force login